bedrock/security/models.py

```diff
@@ -67,7 +67,7 @@ def save(self, force_insert=False, force_update=False,
 class SecurityAdvisory(models.Model):
     id = models.CharField(max_length=8, primary_key=True, db_index=True)
     title = models.CharField(max_length=200)
-    impact = models.CharField(max_length=100)
+    impact = models.CharField(max_length=100, null=True)
```

Comment:
Also, I'm curious about the difference between `default=''` and `blank=True`. I feel like the latter is the official way to go, but I'm not sure whether there's a significant difference.